src/components/Layout/NavButton.module.css

```
@@ -0,0 +1,21 @@
+.NavButton {
+  font-size: 16px;
+  line-height: 120%;
+  font-weight: bold;
+
+  background: none;
```

Stylelint (17.3.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)